code/espurna/thermostat.h

```diff
@@ -13,6 +13,12 @@ Copyright (C) 2017 by Dmitry Blinov <dblinov76 at gmail dot com>
 
 #if THERMOSTAT_DISPLAY_SUPPORT
 #include <SSD1306.h> // alias for `#include "SSD1306Wire.h"`
+
+// It looks more natural that one click will enable display
+// and long click will switch relay
+#define BUTTON1_CLICK           BUTTON_MODE_DISPLAY_ON
+#define BUTTON1_LNGCLICK        BUTTON_MODE_TOGGLE
```

Comment:
Not showing in CI, as display is off by default:
```
In file included from espurna/pr2173/code/espurna/espurna.ino:62:0:
espurna/thermostat.h:19:0: warning: "BUTTON1_CLICK" redefined [enabled by default]
 #define BUTTON1_CLICK           BUTTON_MODE_DISPLAY_ON
 ^
In file included from espurna/config/all.h:36:0,
                 from espurna/pr2173/code/espurna/espurna.ino:22:
espurna/config/defaults.h:87:0: note: this is the location of the previous definition
 #define BUTTON1_CLICK       BUTTON_MODE_TOGGLE
 ^
In file included from espurna/pr2173/code/espurna/espurna.ino:62:0:
espurna/thermostat.h:20:0: warning: "BUTTON1_LNGCLICK" redefined [enabled by default]
 #define BUTTON1_LNGCLICK        BUTTON_MODE_TOGGLE
 ^
In file included from espurna/config/all.h:36:0,
                 from espurna/pr2173/code/espurna/espurna.ino:22:
espurna/config/defaults.h:162:0: note: this is the location of the previous definition
 #define BUTTON1_LNGCLICK    BUTTON_MODE_RESET
 ^
/
```

This should probably be in dependencies.h, so we already have these values after all.h is loaded

Reply:
Moved button definitions into the right place. Checked on default configuration and with display support enabled. Please verify that it don't cause conflicts any more. Please note, that thermostat support should be enabled also in this case as display require it.

Reply:
Not seeing any more issues with code/test/build/sensor.h config + display flag, lgtm